docs/pages/api-reference/expressions/now.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Now
+order: 0
+status: published
+---
+### Now
+
+Function to now timestamp as a column, similar to `current_timestamp()` in Polars or Spark.
```

Comment:
The grammar needs to be adjusted + better to reference `datetime.now` as an example. Maybe `Function to get the current timestamp, similar to what datetime.now does in Python`.

Reply:
changed